test/stream_group_test.dart

```diff
@@ -435,6 +435,23 @@ main() {
 
     expect(merged.toList(), completion(unorderedEquals(["first", "second"])));
   });
+
+  test("mergeBroadcast() emits events from all components streams", () {
+    var controller1 = StreamController<String>();
+    var controller2 = StreamController<String>();
+
+    var merged =
+        StreamGroup.mergeBroadcast([controller1.stream, controller2.stream]);
+
+    controller1.add("first");
+    controller1.close();
+    controller2.add("second");
+    controller2.close();
+
+    expect(merged.isBroadcast, isTrue);
+
+    expect(merged.toList(), completion(unorderedEquals(["first", "second"])));
```

Comment:
does this need to be an `unorderedEquals` check? In theory these would always come in the order they are emitted by the given streams right?

Reply:
🤷‍♂ - it duplicates the existing test... I'll followup on this.